src/app/components/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -60,7 +60,11 @@
                 almIcon
                 [iconType]="workItem.attributes['system.state']"
                 tooltip="{{workItem.attributes['system.state']}}"
-                placement="right"></span>
+                placement="right">
+              </span>
+              <span class="pull-left f8-wi__list-description-state-text padding-v-2">
+                {{ workItem.attributes['system.state'] }}
+              </span>
```

Comment:
![state2](https://user-images.githubusercontent.com/1892722/33646351-f2271e0e-da46-11e7-9b1a-cd4f880291cd.png)
![state1](https://user-images.githubusercontent.com/1892722/33646352-f25b4634-da46-11e7-8acd-66f295ada8d2.png)

The text needs to be centre aligned vertically.

Reply:
fixed